hbw/app/assets/stylesheets/hbw/branding.sass

```diff
@@ -1,10 +1,11 @@
 $button-types: default primary danger info success warning
 
-body
+body, .hbw-entity-tools
   font-size: var(--font-size, var(--base-font-size))
 
 .btn
-  padding: var(--btn-padding, var(--base-btn-padding))
+  font-size: var(--font-size, var(--base-font-size))
```

Comment:
This is very strange. Do you want to say we should set `font-size` for every element we want to affect?